targets/TARGET_WICED/WICED.md

```diff
@@ -0,0 +1,229 @@
+# How to add the WICED SDK
+
+This document summarizes how to download and integrate the WICED SDK into Mbed OS.
+
+**NOTE:** This document is intended for when the WICED SDK is extracted from Mbed OS.
+
+There are several files not provided by the WICED SDK. This is a non-exhaustive
+list of files that are currently provided by Mbed OS:
+- w_generated_mac_address.txt
+- w_generated_security_dct.h
+- w_resources.h
+- mbed_lib.json
+
+## Download the WICED SDK
+
+You can find the full instructions to download the WICED SDK with Advantech's patches
+on Advantech's Wiki:
+http://ess-wiki.advantech.com.tw/view/WISE-1530_SDK
+
+To get the WICED SDK, download WICED-Studio-5.2.0 for your platform:
+https://community.cypress.com/community/wiced-wifi/wiced-wifi-documentation
+
+Once downloaded, move the WICED SDK folder into `targets/TARGET_WICED`.
+
+For the WISE1530, download the patch from Advantech:
+`WM-BN-BM-22_SDK_5.1.x_platform_patch.zip`
+
+Once downloaded, copy the contents into `targets/TARGET_WICED/WICED`.
+
+## Modify the WICED SDK to support Mbed OS's build tools
+
+1. Remove these files because they are unnecessary and large:
+   - `targets/TARGET_WICED/WICED/tools`
+   - `targets/TARGET_WICED/WICED/build`
+   - `targets/TARGET_WICED/WICED/apps`
+   - `targets/TARGET_WICED/WICED/factory`
+   - `targets/TARGET_WICED/WICED/doc`
+   - `targets/TARGET_WICED/WICED/RTOS/NoOS`
+   - `targets/TARGET_WICED/WICED/RTOS/NuttX`
+   - `targets/TARGET_WICED/WICED/RTOS/ThreadX`
+   - `targets/TARGET_WICED/WICED/network/NetX`
+   - `targets/TARGET_WICED/WICED/network/NetX_Duo`
+   - `targets/TARGET_WICED/WICED/network/NoNS`
+   - `targets/TARGET_WICED/WICED/network/NuttX_NS`
+
+1. Add the `w_` prefix to all c/h files in WICED.
```

Comment:
2.

Reply:
Markdown automatically numbers this formatting. See:
https://github.com/geky/mbed/blob/fe39a2bf5064cb2b3d2a8f3e39b74beef23b2d02/targets/TARGET_WICED/WICED.md